base/redactable_error.go

```diff
@@ -20,7 +20,8 @@ func RedactErrorf(fmt string, args ...interface{}) *RedactableError {
 
 // Satisfy error interface
 func (re *RedactableError) Error() string {
-	return fmt.Sprintf(re.fmt, re.args...)
+	redactedArgs := redact(re.args)
+	return fmt.Sprintf(re.fmt, redactedArgs...)
```

Comment:
I'm not sure we should be redacting in the Error method. This can appear in various places that aren't logging (HTTP responses, code that compares error values, etc.) and shouldn't be affected by log redaction